### ISSUE TITLE
In a file, "ODM2PythonAPI/ODM2/apiCustomType.py"

### ISSUE THREAD

**Issue by cdesyoun** [ODM2/ODM2PythonAPI#2]
using the postgresql 9.4 + PostGIS 2.1, in my case, "path" variable is changed below:  
    #path = '"odm2"."samplingfeatures"."FeatureGeometry"'  
    path = 'odm2.samplingfeatures.FeatureGeometry'

**Comment by emiliom** [comment]
I agree that the statement is incorrect for postgresql, and @cdesyoun's corrected statement is what it should be. I haven't run into this error yet b/c I'm only just starting with ODM2API, but I can also confirm that [the current source code](https://github.com/ODM2/ODM2PythonAPI/blob/master/src/api/ODM2/apiCustomType.py) (line 34) still needs to be corrected.

(I won't submit PR's until I'm more comfortable with the code, and basic things are working reasonably ok)

**Comment by sreeder** [comment]
@cdesyoun  I have been working on this so that it is generated by the  compiler and no longer statically entered into the code. Can you verify if the new version is still causing you issues?

**Comment by emiliom** [comment]
@sreeder, I can report that I successfully tested most of [Demo.py](https://github.com/ODM2/ODM2PythonAPI/blob/master/src/api/Demo.py), using my PostgreSQL use case db. I made a couple of changes, mainly setting the `session_factory` connection to my own database, of course. I could get a listing of all Variables and People. `read.getSamplingFeatureByCode()` worked, but I'm having some issues with `read.getSamplingFeaturesByType()`. But that's a narrower topic, that I need to look into and is outside the scope of this github issue.

Thanks! Big progress, for sure.
